Documentation/ReleaseNotes.md

```diff
@@ -7,9 +7,21 @@
 
 ### What's new
 
+**Enable MSBuild for Unity removed from the configuration dialog**
+
+To prevent the MRTK configuration dialog from repeatedly displaying when `Enable MSBuild for Unity` is unchecked, it has been moved to the `Mixed Reality Toolkit' menu as shown in the following image.
+
+![MSBuild for Unity menu items](Images/ConfigurationDialog/MSB4UMenuItems.png)
+
+This change also adds the ability to uninstall / disable using MSBulid or Unity.
```

Comment:
Should this just be "remove" MS Build for unity (also, not "or Unity" -> "for Unity")

Uninstalling and disabling are two different concepts and we don't actually support both here.

Reply:
yep. remove is the correct thing and thx for catching the typeo